### ISSUE TITLE
Update URL on parameter change

### ISSUE THREAD

**Issue by zachberry** [rolodato/volca-keys-editor#3]
Might be cool to update the location.hash on parameter change so the user could copy the address to share patches (and a refresh would maintain the same values). If this sounds like something desired I could put in a PR - however I could see wanting to keep the URL as is so refreshing the page maintains the default values.

**Comment by rolodato** [comment]
Yes, I had considered doing this, and it does make sense as a feature. I'd still want to keep the URL displayed on the page itself (in the clear or behind a button, doesn't matter) for fullscreen users that don't have immediate access to the URL bar.

We wouldn't be losing much by implementing this, because the Initialize button resets all parameters to their initial values. If the initial values were determined by a URL patch, this still works.

I am a React newbie so if you find the code confusing or have any suggestions, please let me know. Thanks!
